src/isPalindrome.js

```diff
@@ -19,7 +19,11 @@
  * @return {boolean}
  */
 function isPalindrome(str) {
-  // write code here
+  const alphanumericLowerCaseString = str.replace(/[\W_]/g, '').toLowerCase();
```

Comment:
старайся не давать такие длинные названия
```suggestion
  const replacedString = str.replace(/[\W_]/g, '').toLowerCase();
```